src/relay/transforms/device_domains.cc

```diff
@@ -47,20 +49,22 @@ constexpr size_t mix(size_t h1, size_t h2) {
  * See te_compiler.cc for where this rewriting occurs.
  */
 DeviceCopyProps GetPrimitiveDeviceCopyProps(const CallNode* call_node) {
-  auto tir_call_attrs = call_node->attrs.as<TIRCallAttrs>();
-  if (tir_call_attrs == nullptr) {
-    return {};
-  }
-  if (tir_call_attrs->metadata.count("source_device") != 1 ||
-      tir_call_attrs->metadata.count("dst_device") != 1) {
-    return {};
+  if (call_node->op == CallLoweredOp()) {
```

Comment:
I think this would become the GetLoweredDeviceCopyProps mentioned above, so no need for the helper here.

Reply:
we decided we don't need that actually so this is OK as is.